EnglishFilter/sections/specific.txt

```diff
@@ -8233,7 +8233,7 @@ nudespree.com##.main-menu > li > a[rel="nofollow"]
 vidz72.com##.partnier-top-nav
 forebears.io##.banner-h
 ||forebears.io/data/*/dna-h.html?*&ads=
-biqle.ru##.message-container
+vkz.de,biqle.ru##.message-container
```

Comment:
We already have rules for this type of cookies. Also missed social share.
Fixed https://github.com/AdguardTeam/AdguardFilters/commit/816186f67d83855629cdfd8f780fe4d237137b91